src/format.rs

```diff
@@ -224,9 +230,12 @@ async fn run(
         let start_row = &buf[(ctx.codeblock_start as i32 + offset) as usize];
         let whitespace = utils::get_start_whitespace(start_row);
 
+        let indent = utils::get_start_whitespace(&buf[(ctx.start as i32) as usize]);
```

Comment:
Code wise this is fine, but the naming is a bit weird now. Maybe it should be `codeblock_indent` and `content_indent`?